tests/test_formatter.py

```diff
@@ -203,3 +203,100 @@ def test_escape(self):
 
         # Escaping is atomic: brackets inside brackets are treated literally
         assert self.formatter.format(datetime(1, 1, 1), "[[[ ]]") == "[[ ]"
+
+
+class TestDateTimeFormatterPremadeFormats:
```

Comment:
We recently switched and merged in changes to the tests, mainly moving them all to pytest and using fixtures, can you rebase your changes and have this class use the formatter fixture instead?

Reply:
Thank you for your feedback! Unfortunately, we are having trouble understanding how to go about rebasing our code because we are working in our own branch on a forked repository, while the fixture updates are in the master branch. Do you have any guidance as to how we should go about merging these changes?

Reply:
https://github.com/crsmithdev/arrow/blob/master/tests/test_formatter.py#L12

Put up this decorator on top of the class. and you can remove the setup_class function.